extensions/xcode/src/open-url-in-simulator.command.tsx

```diff
@@ -0,0 +1,37 @@
+import { XcodeSimulatorService } from "./services/xcode-simulator.service";
+import { Clipboard, closeMainWindow, showToast, Toast } from "@raycast/api";
+
+interface OpenSimulatorArguments {
+  url?: string;
+}
+
+export default async (props: { arguments: OpenSimulatorArguments }) => {
```

Comment:
Type of props must be `LaunchProps` otherwise the command will never receive the URL which the user has entered in the argument placeholder.

```ts
export default async (props: LaunchProps<{ arguments: { url?: string } }>) => { ...}
```

Reply:
I did use this at first but for some reason I get a warning on this `LaunchProps` -> `Type 'LaunchProps' is not generic`